### ISSUE TITLE
f.lux: Version number missing when installed

### ISSUE THREAD

**Issue by r33int** [microsoft/winget-pkgs#16003]
Create issues related to the winget.exe client [here](https://github.com/microsoft/winget-cli/issues/new?template=Bug_Report.md)

# Brief description of your issue

The f.lux package, when installed, does not report a version number, whereas the repository has a version number for that package. This results on f.lux always updating when using `winget upgrade --all`, even when the latest version is already installed.

# Steps to reproduce

- Install f.lux using winget
- Run an upgrade with `winget upgrade --all`

# Expected behavior

The package should be reporting a version number, being listed as up to date and not be upgraded.

# Actual behavior

The package is always upgrading, no matter whereas this is necessary or not.

# Environment

```none
[winget --info]
Windows Package Manager v1.0.11451
Windows: Windows.Desktop v10.0.19043.1023
Package: Microsoft.DesktopAppInstaller v1.11.11451.0

Any other software?
```

**Comment by ItzLevvie** [comment]
This is related to https://github.com/microsoft/winget-pkgs/issues/15946#issuecomment-853528280.

The version from `winget --list` relies on data from Control Panel's Add or Remove Programs (ARP) list; the f.lux application currently doesn't report a version number in Control Panel, and therefore, it'll show up in `winget --list` as `unknown` instead.

![image](https://user-images.githubusercontent.com/11600822/120659775-99314b00-c47e-11eb-8cca-8038257b200e.png)

**Comment by TheJanzap** [comment]
Maybe winget could check after a installation or upgrade if a version number is set and if it isn't store it somewhere internally? But I'm unsure if this could lead to slowdown on operations such as `winget list` or `winget upgrade`.

**Comment by Trenly** [comment]
I know it isn't foolproof, but knowing that it pulls data from the ARP list, wouldn't it be possible to have winget set the `DisplayVersion` registry key when installing a program if it is null or empty after the installation? It might take some technical work to figure out which registry entries would need to be modified, but then it wouldn't lead to slowdowns in the list or upgrade functions

I did a quick test with Gyazo, where the version appears as undefined. I added the display version key and set it to the latest version, and after I did that,  Gyazo no longer appeared in the `winget upgrade` list.

**Comment by denelon** [comment]
The ideal solution is to politely request f.lux Software LLC add the version to the installer. The Windows Package Manager is still very new in the world of software, and it's best to allow publishers and Independent Software Vendors (ISVs) time to decide how they would like their product experience to be for their customers.

**Comment by Trenly** [comment]
This will be resolved by #109644
